EiffelEventLinkV1.go

```diff
@@ -0,0 +1,67 @@
+// Copyright Axis Communications AB.
+//
+// For a full list of individual contributors, please see the commit history.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// THIS FILE IS AUTOMATICALLY GENERATED AND MUST NOT BE EDITED BY HAND.
+
+package eiffelevents
+
+// EventLinksV1 represents a slice of EventLinkV1 values with helper methods
+// for adding new links.
+type EventLinksV1 []EventLinkV1
+
+var _ LinkFinder = &EventLinksV1{}
+
+// Add adds a new link of the specified type to a target event.
+func (links *EventLinksV1) Add(linkType string, target MetaTeller) {
+	*links = append(*links, EventLinkV1{Target: target.ID(), Type: linkType})
+}
+
+// Add adds a new link of the specified type to a target event identified by an ID.
+func (links *EventLinksV1) AddByID(linkType string, target string) {
+	*links = append(*links, EventLinkV1{Target: target, Type: linkType})
+}
+
+// FindAll returns the IDs of all links of the specified type, or an empty
+// slice if no such links are found.
+func (links EventLinksV1) FindAll(linkType string) []string {
```

Comment:
You alternate between (links *EventLinks) and (links EventLinks), is this intentional?

Reply:
Yes. Not sure if it was done for a good reason in the old code, but this is just a carry-over.